Sources/BraintreeAmericanExpress/BTAmericanExpressClient.swift

```diff
@@ -7,13 +7,14 @@ import BraintreeCore
 ///  `BTAmericanExpressClient` enables you to look up the rewards balance of American Express cards.
 @objc public class BTAmericanExpressClient: NSObject {
     
-    private let apiClient: BTAPIClient
+    /// exposed for testing
+    var apiClient: BTAPIClient
     
     ///  Creates an American Express client.
-    /// - Parameter apiClient: An instance of `BTAPIClient`
+    /// - Parameter authorization: Your client token or tokenization key
```

Comment:
very nit, take it or leave it:

```suggestion
    /// - Parameter authorization: A valid client token or tokenization key used to authorize API calls
```